gridstatus/pjm.py

```diff
@@ -757,6 +757,9 @@ def get_lmp(
                 ),
             )
 
+        # returns on the latest version of the data
```

Comment:
Does PJM issue corrections for data?

Reply:
i think that is what happened here. they do keep older versions so I think we can backfill corrections in the future if needed